app/controllers/lead_providers/confirm_schools_controller.rb

```diff
@@ -28,14 +28,30 @@ def start
       redirect_to action: :show
     end
 
+    # TODO: This is temporary behaviour and will be replaced with Partnership creation
     def update
```

Comment:
Would this not be `create`?

Reply:
Good question! I'd say it is neither - it is `confirm`. CRUD/resourcefullness  is another part of Rails philosophy that makes very little sense to me:P

Reply:
Yes, this being a `PUT` feels wrong somehow